tutorials/vscodeJavaSetup.md

```diff
@@ -0,0 +1,182 @@
+{% set title="VS Code - Setting up a Java project" %}
+<frontmatter>
+  title: "{{ title }}"
+  pageNav: 2
+</frontmatter>
+
+# {{ title }}
+
+This tutorial will guide you through creating and managing Java projects in VS Code.
+
+<box type="info" seamless>
+
+This tutorial is adapted from the [official VS Code Java Tutorial](https://code.visualstudio.com/docs/java/java-tutorial).
+</box>
+
```

Comment:
Start by mentioning the installation of these first? Best not to assume they are installed already. 
1. JDK (point to our own installation instructions)
2. VS Code